asyncapi.yaml

```diff
@@ -0,0 +1,351 @@
+#From https://github.com/openapi-contrib/style-guides/pull/4/files
```

Comment:
Actually, I feel like this file is supposed to be removed?

Reply:
Yeah this is gonna go away so lets not mention it.

Reply:
@philsturgeon @P0lip That file was only a todolist to me. I dropped it.